src/Training.AirBnb.Clone.Backend/Backend Project/Domain/Services/ListingReviewService.cs

```diff
@@ -0,0 +1,106 @@
+﻿using Backend_Project.Domain.Entities;
+using Backend_Project.Domain.Exceptions.ListingReviewException;
+using Backend_Project.Domain.Interfaces;
+using Backend_Project.Persistance.DataContexts;
+using System.Linq.Expressions;
+
+namespace Backend_Project.Domain.Services
+{
+    public class ListingReviewService : IEntityBaseService<ListingReview>
+    {
+        private readonly IDataContext _appDataContext;
+
+        public ListingReviewService(IDataContext appDataContext)
+        {
+            _appDataContext = appDataContext;
+        }
+
+        public async ValueTask<ListingReview> CreateAsync(ListingReview listingReview, bool saveChanges = true)
+        {
+            if (string.IsNullOrWhiteSpace(listingReview.Comment) || listingReview.Comment.Length < 1000)
+                throw new ListingReviewFormatException("Invalid comment!");
+            if (listingReview.Rating < 0 || listingReview.Rating > 5)
+                throw new ListingReviewFormatException("Invalid rating!");
+            
```

Comment:
Validatsiya uchun alohida private method yaratib usha yerdan bool qaytarish kerak. CreateAsync methodi ichida shu methoddan foydalangan afzal. Single responsibility ga to'g'ri kelmay qolgan.